example/pug-lang/src/types/Subst.c

```diff
@@ -0,0 +1,79 @@
+/* -*- coding: utf-8-unix -*- */
+
+#include "types/Subst.h"
+#include "types/Types.h"
+
+#include <inttypes.h>
+
+// -----------------------------------------------------------------------
+Tyvar freshTVar(void) {
+  static uint32_t n = 0;
+  CharBuff b = {0};
+  mem_printf(&b, "#%" PRIx32, n++);
+  return (Tyvar){.ident = b.data};
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'ident' is assigned a value that is never used.](https://app.codacy.com/manual/mori0091/cparsec3/pullRequest?prid=5549565)

Reply:
No need to fix.